.github/workflows/validate_json.yml

```diff
@@ -16,8 +16,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - name: Compile yaml schema to JSON
+        run: |
+          pip install pyyaml
+          python -c 'import json, yaml, sys; json.dump(yaml.safe_load(sys.stdin), sys.stdout, indent=2)' < AgContext.yaml > AgContext.json
```

Comment:
This should be extended to cover all the YAML files.

There should be a main schema file for the overall COCO document, and we can use that in the validator below. Let me know if I can support you in doing this.